_commands/basics/less.md

```diff
@@ -3,22 +3,69 @@
 
 less
 --
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+`less` is a tool to view a text file one screen at a time. You can navigate through the file with `j` or `RETURN`, and backwards with `k`
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ less hello.txt
+# displays hello.txt
+q # to quit
 ~~~
 
-<!--more-->
+<!--less-->
```

Comment:
This needs to stay as "more" because the templating checks for this comment to decide where to split the short reference page example and the fuller examples.